django_freeradius/base/base.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from model_utils.fields import AutoLastModifiedField
+
+
+class TimeStampedEditableAdmin(models.Model):
+
+    # An abstract base class model that provides self-updating
+    # modified  fields.
+
+    id = models.UUIDField(primary_key=True, editable=False)
```

Comment:
As discussed, probably ``UUIDField`` is not useful for freeradius-native models. And it's definitely too "heavy" (sizewise) for highly populated tables like ``AbstractRadiusAccounting``
I would keep this base model without the ``id`` and I'd just use standard numeric primary keys
I would also add a ``created = AutoCreatedField(_('created'))``
Check the class name too (it should be ``TimeStampedEditableModel``)